### ISSUE TITLE
License of this repo

### ISSUE THREAD

**Issue by planger** [thegecko/vscode-theia-extension#1]
Hi @thegecko, thank you for providing this example repo for adding custom VS Code extension API in Theia. It is certainly very useful. Would you be OK to license this work under the same OSS license as Theia (EPL-2.0, GPL-2.0)?

Btw, I did a quick check whether it still runs with the latest Theia release and found it to be working fine:
https://github.com/eclipsesource/vscode-theia-extension/tree/update-1.36.0
I'm happy to open a PR for this update, if you like.

**Comment by thegecko** [comment]
Yeah, no problem. We can also transfer this to `eclipse-cdt-cloud` or `eclipse-theia` if you like?

**Comment by planger** [comment]
Great, thanks! I guess `eclipse-theia` would be a perfect fit, since it is also referenced from the [Theia docs](https://theia-ide.org/docs/tips/). Do you plan to trigger the transfer or is there anything I can help with?
